packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx

```diff
@@ -422,10 +422,14 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
 
   private _getAriaLabel = (point: IChartDataPoint): string => {
     const legend = point.xAxisCalloutData || point.legend;
+    const benchMark = point.data;
```

Comment:
benchmark is an optional property. Announce the benchmark only if present.

Reply:
Added condition check in next line  " + (benchMark ? 'with benchmark at ' + `${benchMark}, ` : '')"